frappe/email/doctype/email_account/test_email_account.py

```diff
@@ -178,10 +178,11 @@ def test_threading_by_message_id(self):
 		event = frappe.get_doc(dict(doctype='Event', subject='test-message')).insert()
 
 		# send a mail against this
-		frappe.sendmail(recipients='test@example.com', subject='test message for threading',
-			message='testing', reference_doctype=event.doctype, reference_name=event.name)
+		from frappe.core.doctype.communication.email import make
+		make(doctype=event.doctype, name=event.name, subject='test message for threading',
+		     content='testing',recipients='test@example.com', send_email=True)
 
-		last_mail = frappe.get_doc('Email Queue', dict(reference_name=event.name))
+		last_mail = frappe.get_doc('Communication', dict(reference_name=event.name))
 
```

Comment:
in "Daily Work Summary" threading is by email queue as there are no communications for sent emails